docs/types/conversion.rst

```diff
@@ -99,6 +99,27 @@ rules explicit::
     uint8 d = uint8(uint16(a)); // d will be 0x34
     uint8 e = uint8(bytes1(a)); // e will be 0x12
 
+``bytes`` arrays and ``bytes`` calldata slices can be converted explicitly to fixed bytes types (``bytes1``/.../``bytes32``).
```

Comment:
I think we usually list the allowed explicit conversions in the sections about the individual types. Can you check and maybe add something there as well?

Reply:
I couldn't find anything for ``bytes`` but will check once more